isaric/parsers/ccp-netherlands.toml

```diff
@@ -0,0 +1,736 @@
+
+  [visit.treatment_antibiotics]
+    field = "Antibiotic"
+    ref = "Y/N"
+
+  [visit.treatment_inhaled_nitric_oxide]
+    combinedType = "any"
+    fields = [
+      { field = "patient_interventions#Inhaled_Nitric_Oxide", values = { 1 = true, 0 = false } },
+      { field = "Inhaled_Nitric_Oxide", ref = "Y/N" },
+    ]
+
+  [visit.treatment_noninvasive_ventilation]
+    combinedType = "any"
+    fields = [
+      { field = "patient_interventions#Noninvasive_ventilation_eg_BIPAP_CPAP", values = { 1 = true, 0 = false } },
+      { field = "Non_invasive_ventilation", ref = "Y/N" },
+    ]
+
+  [visit.treatment_high_flow_nasal_cannula]
+    field = "patient_interventions#Highflow_nasal_canula_oxygen_therapy"
+    values = { 1 = true, 0 = false }
+
+  [visit.treatment_cardiovascular_support]
+    combinedType = "any"
+    fields = [
+      { field = "patient_interventions#Extra_corporeal_life_support_ECLS", values = { 1 = true, 0 = false } },
+      { field = "Extracorporeal_support", ref = "Y/N" },
+      { field = "patient_interventions#Any_vasopressorinotropic_support", values = { 1 = true, 0 = false } },
+      { field = "Inotropes_vasopressors", ref = "Y/N" },
+    ]
+
+  [visit.treatment_neuromuscular_blocking_agents]
+    field = "patient_interventions#Neuromuscular_blocking_agents"
+    values = { 1 = true, 0 = false }
+
+  [visit.treatment_other]
+    combinedType = "set"
+    excludeWhen = "none"
+    fields = [{ field = "Specify_other_intervention_procedure" }]
+
+  [visit.outcome]
+    field = "Outcome"
+
+    [visit.outcome.values] # check correspondence of these values
+      1 = "discharged"
+      2 = "hospitalised"
+      3 = "transferred"
+      4 = "death"
+      5 = "palliative discharge"
+
+[visit.date_outcome]
+  field = "Outcome_dt"
+
+  # OBSERVATION
+  # required fields - phase, date, name
+
+[[observation]]
+  name = "avpu"
+  date = { field = "assessment_dt" }
+  phase = "study"
+  text = { field = "AVPU", values = { 1 = "Alert", 2 = "Verbal", 3 = "Pain", 4 = "Unresponsive" } }
+
+[[observation]]
+  name = "pneumonia"
+  start_date = { field = "admission_dt" }
+  date = { field = "Outcome_dt" }
+  duration_type = "event"
+  phase = "study"
+  is_present = { field = "Viral_pneumonitis_1", ref = "Y/N" }
+  context = ['viral']
+
+[[observation]]
+  name = "pneumonia"
+  start_date = { field = "admission_dt" }
+  date = { field = "Outcome_dt" }
+  duration_type = "event"
+  phase = "study"
+  is_present = { field = "Bacterial_pneumonia_1", ref = "Y/N" }
+  context = ['bacterial']
+
+[[observation]]
+  name = "pneumonia"
+  start_date = { field = "admission_dt" }
+  date = { field = "Outcome_dt" }
+  duration_type = "event"
+  phase = "study"
+  is_present = { field = "COP_1", ref = "Y/N" }
+  context = ['COP']
+
+[[observation]]
+  name = "diastolic_blood_pressure_mmHg"
+  date = { field = "admission_dt" }
+  phase = "admission"
+  value = { field = "dias_bp" }
+  if.all = [
+    { dias_bp = { "!=" = -96 } },
```

Comment:
these values (-96, -98, -99) here and below - are they arbitrary or specific to the attributes?

Reply:
They seem to be arbitrary null/not done values that are scattered around, anything between -95 and -99 has been used in various places.